docs/contributing/ui-testing.md

```diff
@@ -0,0 +1,102 @@
+---
```

Comment:
I hope that the link below will also be included in somewhere of this document.
https://github.com/enactjs/ui-test-utils